### PR TITLE
WIP: Add date field import files

### PR DESCRIPTION
**Pull request by afranke**
Import files represent information at a given date. This will help track this date.

:warning: untested because of permission errors with a test database.

### CODE REVIEW

File: matcher/dashboard/templates/imports/show.html [hunk 1]
```diff
@@ -26,6 +26,9 @@
               {% endif %}
             </dd>
 
+            <dt>Effective date</dt>
+            <dd>{{ file.effective_date() }}</dd>
```
Comment (sandhose) [inline matcher/dashboard/templates/imports/show.html]: shouldn't be a function call

File: matcher/migrations/versions/2a72a6b54510_.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+"""empty message
```
Comment (sandhose) [inline matcher/migrations/versions/2a72a6b54510_.py]: please provide a meaningful migration message

**Comment by afranke** [comment]
And I see the migration doesn’t actually work (incorrectly accessing columns) thanks to the CI. I’m on it.

File: matcher/migrations/versions/2a72a6b54510_.py (hunk 2 above)
Comment (afranke) [reply matcher/migrations/versions/2a72a6b54510_.py]: Oops. Missed that one. Fixed.

File: matcher/dashboard/templates/imports/show.html (hunk 1 above)
Comment (afranke) [reply matcher/dashboard/templates/imports/show.html]: Indeed. I would blame whiskey except I’m sober. Let’s blame lack thereof. Fixed.

**Comment by afranke** [comment]
> And I see the migration doesn’t actually work (incorrectly accessing columns) thanks to the CI. I’m on it.

Fixed too. Do we need a `# noqa` to make CI happy?

**Comment by afranke** [comment]
Tested on the staging environment, seems to work as advertised. http://scraping-staging.obs.coe.int/imports/14 is a good example with a workflow that spans several days and it shows that the upload date is the one taken into account. We also checked with a freshly imported file (after the deployment), so not one where the migration is used, and it works as well.

**Pull request merged**